src/System.Net.Http.WinHttpHandler/tests/FunctionalTests/WinHttpHandlerTest.cs

```diff
@@ -5,12 +5,23 @@
 using System.Net;
 using System.Net.Http;
 using System.Net.Tests;
-
+using System.Threading;
+using System.Threading.Tasks;
 using Xunit;
 using Xunit.Abstractions;
 
 namespace System.Net.Http.WinHttpHandlerTests
```

Comment:
@josguil Look at any of the other tests in System.IO, System.Net.Primitives, System.Net.Http.
The proposal is to change `System.Net.Http.WinHttpHandlerTests` to `System.Net.WinHttpHandler.Tests`.